### PR TITLE
Gb/disc pressure bug

### PR DESCRIPTION
**Pull request by grantbuster**

### CODE REVIEW

File: farms/disc.py [hunk 1]
```diff
@@ -70,7 +67,7 @@ def disc(ghi, sza, doy, pressure=101325, sza_lim=87):
 
     AM = (1. / (np.cos(np.radians(Ztemp))
                 + 0.15 * (np.power((93.885 - Ztemp), -1.253)))
-          * pressure / 101325)
+          * 100 * pressure / 101325)
```
Comment (MRossol) [inline farms/disc.py:70]: Whats the 100 * for?

**Review by MRossol — COMMENTED** [review]
I'm a bit confused as to what the input units are vs what the units used on line 68 are.

File: farms/disc.py (hunk 1 above)
Comment (MRossol) [reply farms/disc.py:70]: Ah, gotcha! F'ing confusing man. Okay so all inputs are in hPa/mbar but DISC assumes Pa, got it now.

Comment (grantbuster) [reply farms/disc.py:70]: Exactly and previously disc just wanted Pa as input and did some stupid implicit scaling (rest did something similar) if it thought the units were bad but now its just all in mbar and its stated in the docstrings so if you input something wrong its your own fault haha

Comment (MRossol) [reply farms/disc.py:70]: Read the Docs people otherwise garbage in/garbage out!

**Pull request merged**